# A custom message combining a Pose and a boolean flag for stopping at that coordinate
Header Header
bool stop_at_waypoint
geometry_msgs/Pose pose